lib/mysql.js

```diff
@@ -459,6 +459,22 @@ MySQL.prototype.ping = function(cb) {
   this.execute('SELECT 1 AS result', cb);
 };
 
+MySQL.prototype.buildExpression = function(columnName, operator, columnValue, propertyValue) {
+  if (operator === 'regexp' && typeof columnValue === 'string') {
```

Comment:
Assuming the normalization of the operand of `regexp` is done in juggler, we can clean up the code here.